apps/src/p5lab/spritelab/defaultSprites.json

```diff
@@ -1,71 +1,7 @@
-    "45b457bb-5b22-48f7-8664-4b38957d7799",
-    "d15af884-fb27-455f-b73a-899d07ce8975",
-    "2e9c703b-d8cd-44e5-8cf2-c0d5940b84dc",
-    "b287e053-266b-4eaa-ad59-310083ce3398",
-    "986fdd82-f8e0-4674-806c-62d18f0ca9c3",
-    "0a84474a-a327-47c9-8377-c1c478001cec",
-    "7ecbad62-f575-48eb-8861-b42dbf62d545",
-    "9b98c484-57b8-40cc-ad45-1d1ed02336d6",
-    "1124b3e0-1430-451f-a8b2-137ce8091759",
-    "2c93ddd7-1058-4df8-b4a1-73c1f70a1772",
-    "7887ec4e-1a27-4bfe-898c-3049b131cbe4",
-    "f1c617e5-4a3f-4f04-90f0-7e0305df84af",
-    "8545eeec-aa1c-4193-9896-30e93f7af76b",
-    "fa2a0e9e-e38d-40ee-89a0-cdb72a1fa6cc",
-    "fc4aa65d-fb3c-40af-9845-786503764ad3",
-    "a1d0480c-1263-4847-b188-c892024f6bb8",
-    "a5cf7f32-903e-4d01-b163-08e61c9d39aa",
-    "dfe7d8cc-9097-4fae-a02c-d1951aaf32dc",
-    "98b0bab9-36ca-43b1-8370-c520949c5ec4",
-    "a2110245-266a-42ab-b8dc-84b4165924b5",
-    "f755eca1-9df6-4723-bd2e-0176c7ee1cfd",
-    "d197b759-38ef-4973-a43b-4a07653f62b7",
-    "8c1a5163-04a8-4d78-9886-dc2ca8a2d536",
-    "20c00311-89c5-4eeb-98c9-3edb5ebad13e",
-    "6fa37cd5-4ca2-417a-a603-a3a66e0446be",
-    "8ec43d69-3ae4-4a5f-867d-41bc549a5348",
-    "2256652a-d12f-433f-a8d7-ba1db9869aca",
-    "b16c1f8d-cbf3-4d00-b597-547a4aafb909",
-    "7ae82ac5-bf20-4627-9575-574e29c241c0",
-    "fbf2ef19-964e-4593-9822-9eaf8b8e187f",
-    "5dd1aea2-1f2d-4b7a-90eb-9895b2fc3608",
-    "50274781-8652-48d9-b5ad-03c22efe835a",
-    "7a423fba-22b9-42f0-bf37-5ebc5cdf525e",
-    "da54705b-2478-480b-bda3-758d6c348f3b",
-    "9b89d959-be06-45ed-af41-6861b1288e38",
-    "41f01d32-2495-486b-9a58-b9a188e5ce73",
-    "9deaf802-3ed1-4e9f-ac5b-0a8fd6985c13",
-    "dbfa88e2-0c41-48a6-9047-4b4295cfe73f",
-    "b707e334-a22a-43bb-87eb-035ed95bc104",
-    "5b94a14f-5bb3-432d-9594-fc724897c840",
-    "be61640e-babf-45ca-84d7-9bbf6c8295db",
-    "22e9fe74-0bf7-46aa-9d19-94d0eb00381b",
-    "749d51c7-59ce-4c94-bc4f-12c5234d4a29",
-    "92997275-4280-46dd-9672-4a31de5913b9",
-    "5b430010-3b8f-4e86-a1e1-468d229416d3",
-    "f34f4a47-214f-4e38-94d0-dd07cb2b1dcd",
-    "68917b8c-768a-407e-8258-1e6ebbb2340e",
-    "74539f4c-9f4e-4229-aad7-2650454c3e8a",
-    "08aa254e-653d-4ca0-b943-fc2233db3cde",
-    "20c80bcb-26c2-4701-b3ce-9d8dbe96d459",
-    "a8dabd2a-6faf-4ab5-b870-ee9540d256dc",
-    "12eda44b-58ef-4696-9d37-06623620383b",
-    "8ca751af-ef34-4fd4-9e96-6e985f93f4c2",
-    "d0b3ee56-4324-426a-bd7c-72ea9aec2755",
-    "90ce56d0-f5f9-45ec-a012-32e7e2f57e15",
-    "4b4e5d34-d8d3-4c1a-9d41-e94193afb092",
-    "bd424603-10eb-4e88-8160-4f5e8d1b0e59",
-    "62326fd3-5701-4f32-ac89-c5520fa2430a",
-    "1acf1c10-82d5-42f2-b5f6-9462be53f87a",
-    "9f985df8-6ac3-417f-bcae-6f19e26246df",
-    "918cc09b-0fea-41b7-a64f-ea8fc3d427c9",
-    "1d88dc5d-48f6-42fa-b0e6-a2f18ce9f811",
-    "d6c15c0b-6556-4a40-b9cd-245c53bfdad2",
-    "3d97df38-3c67-497b-937f-48061076fe79"
+    "d15af884-fb27-455f-b73a-899d07ce8975"
   ],
   "propsByKey": {
     "2223bab1-0b27-4ad1-ad2e-7eb3dd0997c2": {
```

Comment:
Realizing I don't actually know... What is this first property for?

Reply:
To clarify, do you mean "frameDelay"?

Reply:
I actually mean this whole set of properties associated with whatever `2223bab1-0b27-4ad1-ad2e-7eb3dd0997c2` refers to. For animations, we usually have properties like `name`, `sourceUrl`,  etc. but this object just has `version` and `categories`.

Reply:
Could it be that you're looking at a compressed version of the changes? When I expand my view to see all the lines between lines 71 and 83, I can see the rest of the information for `2223bab1-0b27-4ad1-ad2e-7eb3dd0997c2` (`bear`, url, framesize, etc.)

Reply:
Oh geez, that was it! I was actually looking at line 7 and lines 17-18 of the new file, but lines 8-16 were compressed. :face_palm: